modules/example/src/smithy4s/example/AgeFormat.scala

```diff
@@ -7,7 +7,7 @@ object AgeFormat extends smithy4s.ShapeTag.Companion[AgeFormat] {
   val id: smithy4s.ShapeId = smithy4s.ShapeId("smithy4s.example", "ageFormat")
 
   val hints : smithy4s.Hints = smithy4s.Hints(
-    smithy.api.Trait(Some("integer"), None, None, None),
+    smithy.api.Trait(Some("integer"), None, None),
```

Comment:
This change is caused by the difference between `1.22.0` and `1.21.0-rc1`. Once we update to smithy 2 (the real one), this will change back.